views/layout.ejs

```diff
@@ -69,23 +69,24 @@
     <script src="<%= protocol %>//cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.6.0/clipboard.min.js"></script>
     <script src="<%= protocol %>//cdnjs.cloudflare.com/ajax/libs/jquery-modal/0.9.1/jquery.modal.min.js"></script>
     <script src = "<%= protocol %>//cdn.jsdelivr.net/simplemde/latest/simplemde.min.js"></script>
-    <script type="text/x-mathjax-config">
-      MathJax.Hub.Config({
-        messageStyle: "none",
-        jax: ["input/TeX","output/CommonHTML"],
```

Comment:
How did you translate these to the new settings below?

Reply:
Using the converter linked above

Reply:
Woops, missed that, thanks